xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -168,7 +168,7 @@ struct UserProfileStatistics: Equatable, Codable, Hashable {
 enum UserCategory: Equatable, Codable, Hashable, CaseIterable {
     case human
     case geist
-    case you
+    case ourself
```

Comment:
Sorry for the noise on this, I think it's better to match the general convention of "ours" instead of "you"

Reply:
Yup, I like it.